src/helpers/assistant/assistant.ts

```diff
@@ -0,0 +1,169 @@
+import { AIMessage, BaseMessage, HumanMessage } from "@langchain/core/messages";
+import { ChatOpenAI } from "@langchain/openai";
+import { END, StateGraph, StateGraphArgs, START, MemorySaver, CompiledStateGraph, messagesStateReducer } from "@langchain/langgraph/web";
+import { ChatPromptTemplate, MessagesPlaceholder, SystemMessagePromptTemplate } from "@langchain/core/prompts";
+import { ToolNode } from "@langchain/langgraph/prebuilt";
+import { RunnableConfig } from "@langchain/core/runnables";
+import { StructuredTool } from "@langchain/core/tools";
+
+import MyDataHelps, { Guid, ParticipantInfo } from "@careevolution/mydatahelps-js";
+
+import {
+    PersistParticipantInfoTool,
+    QueryAppleHealthActivitySummariesTool,
+    QueryAppleHealthWorkoutsTool,
+    QueryDailySleepTool,
+    QueryDeviceDataV2AggregateTool,
+    QueryDeviceDataV2Tool,
+    QueryNotificationsTool,
+    QuerySurveyAnswersTool,
+    QueryDailyDataTool,
+    GetAllDailyDataTypesTool,
+    GetEhrNewsFeedPageTool,
+    GetDeviceDataV2AllDataTypesTool
+} from "./tools";
+
+
+export interface IAssistantState {
+    messages: BaseMessage[];
+    participantInfo: string;
+}
+
+export class MyDataHelpsAssistant {
```

Comment:
Would call this MyDataHelpsAIAssistant and rename the file / folder to be consistent with "AI Assistant"